changelog/0043-optionally-configure-auditcare-db.yml

```diff
@@ -0,0 +1,42 @@
+title: Optionally configure auditcare db
+key: optionally_configure_auditcare_db
+date: 2021-03-23
+optional_per_env: yes
+min_commcare_version:
```

Comment:
Should we add a revision here? Maybe https://github.com/dimagi/commcare-hq/pull/29377/commits/87ac2df454b4bcf0ebd623dbd208f014eba546b0 (the last commit of https://github.com/dimagi/commcare-hq/pull/29377)?

Reply:
```suggestion
min_commcare_version: 87ac2df454b4bcf0ebd623dbd208f014eba546b0
```

Reply:
Eh, I think this is wrong. Maybe we want to set `max_commcare_version` instead? Or maybe not at all. I'm not sure.